tests/backends/fixtures.py

```diff
@@ -22,7 +22,8 @@
 import pytest
 
 from pymc.backends import base
-from pymc.tests import models
+
+from .. import models
```

Comment:
You can still use absolute paths no? like `from tests import models`

That would be more readable.

Reply:
I was worried about `tests` conflicting with a Python package, but you are right, that should be possible.